test/test.coffee

```diff
@@ -562,10 +596,11 @@ describe "StoreIt!", ->
             ).should.throw(StoreitError)
 
         describe "then once we call load()", ->
-            beforeEach ->
+            beforeEach (done)->
```

Comment:
Function arrows (-> and =>) must be spaced properly<br>Line contains inconsistent indentation